### DISCUSSION TITLE
Form Submit Not Working After Auth Update

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#334] [Help Me!] [answered]
Dani Crosby

I updated my Nav/Auth logic - it is working but now my add form submit button is not working. No errors in console so I don't know what to google. I've tweaked a bunch of things, researched tickets, timeboxing myself at an hour :)

PR
https://github.com/danicrosby/LUCID-Front-End-Capstone/pull/69

Index
https://github.com/danicrosby/LUCID-Front-End-Capstone/blob/main/src/App/index.js

Nav
https://github.com/danicrosby/LUCID-Front-End-Capstone/blob/main/src/components/NavBar.js

Form
https://github.com/danicrosby/LUCID-Front-End-Capstone/blob/main/src/components/DreamForm.js

**Comment by drteresavasquez** [comment]
I hope you did this on another branch since you'd hit MVP already 👀. What specific changes are you trying to do and what is their purpose?

**Comment by danicrosby** [comment, marked as answer]
Turns out this error wasn't related to my auth update at all. I had forgotten I moved a few things around when I was styling. I ended up taking my 'add submit' button out of the form itself and placed it in the footer of the card (cause it was pretty).  So lesson learned! Keep vital functions IN the form! 

My workaround, to keep my styling pretty, I moved the form tag outside the body and outside the footer. :)
